code/modules/mob/dead/observer/observer.dm

```diff
@@ -544,6 +544,23 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 	if(!istype(target))
 		return
 
+	if(!mind.original)
```

Comment:
If you wanted admins to still be able to scan themselves, pretty sure you could just add `|| client?.admin_holder?.rights & R_MOD` here.

Otherwise looks fine to me.

Reply:
Yup, I'll throw that in in a bit